src/hooks/dialogmote/document/useReferatDocument.ts

```diff
@@ -42,10 +42,129 @@ export const useReferatDocument = (
   const navbruker = useNavBrukerData();
   const { data: veilederinfo } = useAktivVeilederinfoQuery();
   const isEndringAvReferat = mode === ReferatMode.ENDRET;
-  const { getVirksomhetsnavn, getHilsen } = useDialogmoteDocumentComponents();
-
+  const { getVirksomhetsnavn } = useDialogmoteDocumentComponents();
+  const { malform } = useMalform();
+  const referatTexts = getReferatTexts(malform);
+  const commonTexts = getCommonTexts(malform);
+  const hilsenParagraph = createParagraph(
+    commonTexts.hilsen,
+    veilederinfo?.navn || "",
+    `NAV`
+  );
   const personident = useValgtPersonident();
 
+  const intro = (): DocumentComponentDto[] => {
```

Comment:
Blir en litt stor diff her, fordi jeg flyttet alle utils-metodene inn i hooken for å slippe å måtte sende målform ned til alle. Bruker da heller teksten direkte fra linje 47.